safe/impact_function/impact_function_utilities.py

```diff
@@ -3,6 +3,7 @@
 """Utilities for Impact Function."""
 
 from PyQt4.QtCore import Qt
+from os.path import exists
```

Comment:
I think we should put the core package first before PyQt4

Reply:
Ahah, yes ;-)
https://github.com/inasafe/inasafe/blob/develop/safe/impact_function/impact_function.py#L5
```
Builtins imports
Builtins from imports

Third party imports
Third party from imports

Local imports
Local from imports
```